### PR TITLE
Implemented inspector socket look up

### PR DESCRIPTION
**Pull request by RedMickey**
Implementation of inspector socket look up functional is based on [vscode-js-debug extension](https://github.com/microsoft/vscode-js-debug) code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -679,9 +679,11 @@
 		"quick-smoke-tests": "cd test/smoke/vscode/test/smoke && yarn mocha --skip-setup"
 	},
 	"dependencies": {
+		"@types/ip": "^1.1.0",
```
Comment (SounD120) [inline package.json]: Typescript typings should be installed in `devDependencies`

File: src/cdp-proxy/reactNativeCDPProxy.ts [hunk 2]
```diff
@@ -31,11 +23,24 @@ export class ReactNativeCDPProxy {
...
 
+    private server: Server | null;
+    private hostAddress: string;
+    private port: number;
+    private debuggerTarget: Connection;
+    private applicationTarget: Connection;
+    private logger: OutputChannelLogger;
+    private logLevel: LogLevel;
+    private firstStop: boolean;
+    private dbgEndpointHelper: DebuggerEndpointHelper;
```
Comment (SounD120) [inline src/cdp-proxy/reactNativeCDPProxy.ts]: Why not just `debugEnpointHelper`?

Comment (RedMickey) [reply src/cdp-proxy/reactNativeCDPProxy.ts]: `debugEnpointHelper` consists of 22 letters. It seems to be too long name, so I decided to shorten it a bit.

Comment (SounD120) [reply src/cdp-proxy/reactNativeCDPProxy.ts]: Not sure if this is the right place to shorten. Other variables are written without any shortenings.

**Pull request merged**